app/uk/gov/hmrc/excisemovementcontrolsystemapi/models/auth/AuthorizedRequest.scala

```diff
@@ -16,9 +16,16 @@
 
 package uk.gov.hmrc.excisemovementcontrolsystemapi.models.auth
 
+import generated.IE815Type
 import play.api.mvc.{Request, WrappedRequest}
 
 case class AuthorizedRequest[A](
```

Comment:
Could we rename these request classes based on what they are storing i.e. AuthorizedRequest becomes EnrolmentRequest and AuthorizedIE815Request becomes ParsedXmlRequest or something similar?

Reply:
done